src/components/TodoList.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onRemove: (todoId: number) => void
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  onRemove,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map((todo) => (
+        <>
+          <div
+            key={todo.id}
```

Comment:
the top-level element is the fragment, not the <div>
so `key` should belong to the fragment
```suggestion
        <React.Fragment key={todo.id}>
          <div
```

Reply:
Done!